modules/placement-driver/src/test/java/org/apache/ignite/internal/placementdriver/PlacementDriverTest.java

```diff
@@ -356,6 +404,37 @@ private void publishLease(Lease lease) {
                 noop()
         );
 
-        revisionTracker.waitFor(rev + 1).join();
+        long expRev = rev + 1;
+
+        assertThat(revisionTracker.waitFor(expRev), willCompleteSuccessfully());
+
+        return expRev;
+    }
+
+    private CompletableFuture<PrimaryReplicaEventParameters> listenReplicaBecomePrimaryEvent() {
+        var eventParametersFuture = new CompletableFuture<PrimaryReplicaEventParameters>();
+
+        placementDriver.listen(REPLICA_BECOME_PRIMARY, (parameters, exception) -> {
+            eventParametersFuture.complete(parameters);
```

Comment:
I would also suggest to not ignore the possible exception

Reply:
fix it